pkgs/top-level/all-packages.nix

```diff
@@ -22152,9 +22152,7 @@ in
   audacious = libsForQt5.callPackage ../applications/audio/audacious { };
   audaciousQt5 = audacious;
 
-  audacity-gtk2 = callPackage ../applications/audio/audacity { wxGTK = wxGTK31-gtk2; };
-  audacity-gtk3 = callPackage ../applications/audio/audacity { wxGTK = wxGTK31-gtk3; };
-  audacity = audacity-gtk2;
+  audacity = callPackage ../applications/audio/audacity { };
```

Comment:
Audacity says that gtk2 must be used, "as gtk3 is not yet fully supported". This is from the "building on linux" page I linked in this commit description.